src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import PropTypes, { shape } from "prop-types";
+import { Form, Button } from "react-bootstrap";
+
+export const LoginView = ({ onLoggedIn }) => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (event) => {
+    // this prevents the default behavior of the form which is to reload the entire page
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password,
+    };
+    fetch(
+      `https://movies-flix-al-f68cdd84f041.herokuapp.com/login?Username=${data.Username}&Password=${data.Password}`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("Login response: ", data);
+        if (data.user) {
+          localStorage.setItem("user", JSON.stringify(data.user));
+          localStorage.setItem("token", data.token);
+          onLoggedIn(data.user, data.token); // quick fixed
```

Comment:
A. onLoggedIn was called with two **arguments** (parameters & arguments) don't match. This is causing the token not to be set on main-view component thus not displaying the list of movies after a successful login. The function on main-view component needs to be refactored.

Reply:
That's still happening after I remove the data.token, but this time it's happening only in the initial login, then It's not happening anymore, that's why I left the window.location.reload() function there, so that resolves the problem